### PR TITLE
Add ethnicity field to Confirm Details screen

### PR DESCRIPTION
**Pull request by bryangaledxw**
# Changes in this PR

This PR first extracts person details into a shared component for use on both the Apply confirm and check your answers screens, then adds the new ethnicity field.

This also sneaks in making our capitalisation of nDelius consistent

### Before
localhost_3000_applications_new__csrf=82oq7lli-pjVZ3q4Mzu1snwW5pRRaP1v8R-8 (1)

### After
localhost_3000_applications_new__csrf=82oq7lli-pjVZ3q4Mzu1snwW5pRRaP1v8R-8

# Release checklist

[Release process documentation](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/edit-v2/4247847062?draftShareId=a1c360ab-bd31-4db1-aae3-7cc002761de9).

## Pre-merge checklist

- [ ] Are any changes required to dependent services for this change to work?
  (eg. CAS API)
- [ ] Have they been released to production already?

## Post-merge checklist

- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to test
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to preprod
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to prod

### CODE REVIEW

**Comment by bryangaledxw** [comment]
[Add `ethnicity` field to Confirm Details screen](https://trello.com/c/rNKao3NJ/1243-add-ethnicity-field-to-confirm-details-screen)

File: cypress_shared/components/personDetails.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  constructor(private readonly person: Person) {
+    super()
+  }
+
+  shouldShowPersonDetails(): void {
+    cy.get('dl[data-cy-person-info]').within(() => {
+      this.assertDefinition('Name', this.person.name)
+      this.assertDefinition('CRN', this.person.crn)
+      this.assertDefinition('Date of birth', DateFormats.isoDateToUIDate(this.person.dateOfBirth, { format: 'short' }))
+      this.assertDefinition('NOMS number', this.person.nomsNumber)
```
Comment (tahb) [inline cypress_shared/components/personDetails.ts:15]: I've learnt today that we can no longer assume people have noms numbers and the lack of one should not prevent a referral. I heard from Adam that CAS1 are refactoring things to make this and other bits of oasys risk data optional to work around this.

**Review by tahb — APPROVED** [review]
LGTM. Perhaps something to do around not depending on a noms number being present.

File: cypress_shared/components/personDetails.ts (hunk 1 above)
Comment (bryangaledxw) [reply cypress_shared/components/personDetails.ts:15]: Reckon that's a separate PR? Will merge this in as is, and flag in backlog refinement that making NOMS optional is coming up

**Pull request merged**